app/models/placement_suitability.rb

```diff
@@ -22,6 +22,12 @@ class PlacementSuitability < ApplicationRecord
   validates :address_county, presence: true, length: { maximum: 128 }
   validates :address_postcode, format: { with: /^([A-Z]{1,2}\d[A-Z\d]? ?\d[A-Z]{2}|GIR ?0A{2})$/i, multiline: true }
 
+  def self.search(search)
```

Comment:
we dont need this yet?

Reply:
If we don't need this and there is no test, I'd suggest for it to be removed.

Reply:
yep